analysis/test/src/pointers.rs

```diff
@@ -504,6 +514,7 @@ unsafe fn main_0(mut argc: libc::c_int, mut argv: *mut *mut libc::c_char) -> lib
     test_arg_rec();
     test_shared_ref();
     test_unique_ref();
+    test_ref_field();
```

Comment:
Could you put the new `test_ref_field()` at the end so it doesn't disturb the snapshot as much?  Or is there a reason it's in that position?

Reply:
If you want to, it's done in https://github.com/immunant/c2rust/pull/574.  The diff with `master` is cleaner.